lib/matplotlib/figure.py

```diff
@@ -2103,7 +2103,12 @@ def savefig(self, fname, *, frameon=None, transparent=None, **kwargs):
               `~.backend_pdf.PdfPages`.
             - 'eps' and 'ps' with PS backend: Only 'Creator' is supported.
 
+        pil_kwargs : dict, optional
+            Additional keyword arguments that are passed to Pillow when saving
```

Comment:
Please add the reference `PIL.Image.save()` here as well.

Reply:
done